web/src/components/Error.jsx

```diff
@@ -1,11 +1,11 @@
 import { Center, Heading } from '@chakra-ui/react';
 import useModeColors from '../hooks/useModeColors';
 
-export default function Error({ error = 'something went wrong' }) {
+export default function Error({ error = 'No Data found' }) {
   const { red } = useModeColors();
 
   return (
-    <Center>
+    <Center h="300px">
```

Comment:
Try to not to use px for height or width, it hinders the responsive behavior of application, try to use rem, em or percentage